### PR TITLE
MS. Auct. F. 2. 8: new description

### PR DESCRIPTION
**Pull request by adunning**

### CODE REVIEW

File: collections/Auct_F/MS_Auct_F_2_8.xml [hunk 1]
```diff
@@ -37,49 +36,328 @@
                      <idno type="SCN">8851</idno>
                   </altIdentifier>
                </msIdentifier>
+               <head>The <q>Oxford Virgil</q></head>
                <msContents>
-                  <msItem n="1" xml:id="MS_Auct_F_2_8-item1">
+                  <summary>Works of Virgil with Pseudo-Ovidian argumenta and Carolingian glosses, combining commentators such as <persName key="person_10970700" role="auth">Servius</persName> with interpretations not attested in late antique commentaries but found in other early medieval manuscripts.</summary>
```
Comment (holfordm) [inline collections/Auct_F/MS_Auct_F_2_8.xml:41]: should be "aut"

File: collections/Auct_F/MS_Auct_F_2_8.xml [hunk 2]
```diff
@@ -37,49 +36,328 @@
...
+                        <finalRubric>Explicit liber primus aeneidorum.</finalRubric>
+                     </msItem>
+                     <msItem>
+                        <locus>(fol. 67r)</locus>
+                        <author key="person_285292080">Pseudo-Ovid</author>
+                        <title type="desc">Argument to <title>Aeneid</title> 2</title>
+                        <rubric>Incipit praefatio libri ·ii·</rubric>
+                        <incipit><supplied>F</supplied>unera dardanię narrat flecteque secundo.</incipit>
+                        <explicit defective="true">Somnum quod minitus acceperat haectoris atrum.</explicit>
+                        <bibl type="repertory"><ref target="http://webserver.erwin-rauner.de/php_ancarm/ancarm_refwerke.php?opus=1001&amp;var=ICL&amp;nr_char=2699"><title>ICL</title> 2699</ref></bibl>
```
Comment (holfordm) [inline collections/Auct_F/MS_Auct_F_2_8.xml:140]: these links don't currently work...

File: collections/Auct_F/MS_Auct_F_2_8.xml (hunk 1 above)
Comment (holfordm) [inline collections/Auct_F/MS_Auct_F_2_8.xml:41]: worth referring to Sullivan's article?

File: collections/Auct_F/MS_Auct_F_2_8.xml [hunk 3]
```diff
@@ -37,49 +36,328 @@
                      <idno type="SCN">8851</idno>
                   </altIdentifier>
                </msIdentifier>
+               <head>The <q>Oxford Virgil</q></head>
```
Comment (holfordm) [inline collections/Auct_F/MS_Auct_F_2_8.xml:39]: perhaps better to have something more descriptive? e.g. Works of Virgil with glosses ('The Oxford Virgil'); France (Saint-Germain-des-Prés), 9th century, first half

File: collections/Auct_F/MS_Auct_F_2_8.xml (hunk 2 above)
Comment (adunning) [reply collections/Auct_F/MS_Auct_F_2_8.xml:140]: That is likely due to the `&amp;` URL encoding: <http://webserver.erwin-rauner.de/php_ancarm/ancarm_refwerke.php?opus=1001&var=ICL&nr_char=2699> works for me right now.

**Pull request merged**